lib/class-wp-json-comments-controller.php

```diff
@@ -191,6 +167,115 @@ public function delete_item( $request ) {
 		return array( 'message' => __( 'Deleted comment' ) );
 	}
 
+
+	/**
+	 * Check if a given request has access to read comments
+	 * 
+	 * @param  WP_JSON_Request $request Full details about the request.
+	 * @return bool
+	 */
+	public function get_items_permissions_check( $request ) {
+
+		// If the post id isn't specified, presume we can create
```

Comment:
@rachelbaker doh you are totally right, I misread where you had put this comment. Yes, the use case you mentioned is correct, though the user should probably send a OPTIONS to `wp/comments/?p=123` to check for that specific comment. In which case we should probably return `false` for `POST wp/comments` as I presume someone can't POST there.